pipeline/pipeline.py

```diff
@@ -20,11 +21,14 @@
     while True:
 
         # Starts a timer for each iteration of the pipeline
-        st = time.time()
+        start_time = time.time()
 
         # Fetches all plant data from the api
         plant_api_data = fetch_all_plant_data()
 
+        extract_time = time.time() - start_time
+        print('Time taken to extract data:', extract_time, 'seconds')
```

Comment:
Nice use of print statements to track the time it takes for each step of the pipeline